packages/amplication-server/src/core/billing/billing.service.ts

```diff
@@ -339,6 +342,26 @@ export class BillingService {
     });
   }
 
+  async provisionNewSubscriptionForAwsMarketplaceIntegration(
+    workspaceId: string,
+    planId: BillingPlan,
+    planBillingPeriod: BillingPeriod,
+    addons?: SubscriptionAddon[]
+  ): Promise<void> {
+    if (!this.isBillingEnabled) {
+      return;
+    }
+
+    await this.stiggClient.provisionSubscription({
+      customerId: workspaceId,
+      planId,
+      skipTrial: true,
+      billingPeriod: planBillingPeriod,
+      // promotionCode: "AWS-MARKETPLACE",
```

Comment:
We will need to update this call to create a new subscription telling stigg that the payment is already been taken care of